### DISCUSSION TITLE
Display source blob of .rst files (just like Markdown)

### DISCUSSION THREAD

**Discussion by mhnestler** [community/community#29222] [Other Feature Feedback, Questions, & Ideas]
There is [a button](https://twitter.com/github/status/1443572280924147717) (looks like: `<>`) when viewing a Markdown file, [which adds](https://github.blog/changelog/2021-06-30-parameter-to-disable-markdown-rendering/) `?plain=1` to the URL. This switches from showing the rendered Markdown to showing the raw source, which has advantages like being able to link to line numbers.

The `?plain=1` parameter also works on other rich-text formats such as reStructuredText (.rst), but the button is missing. [Example here.](https://github.com/python/peps/blob/main/pep-0678.rst)

Would love to see this added!

**Comment by mankoff** [comment]
Importantly, the `?plain=1` for Markdown files allows me to click on a line number and open an issue. Manually adding `?plain=1` to an Org file renders with line numbers (thank you! But make it button-able), but when I click on line numbers, I do not get a menu popup.

**Reply by mhnestler** [reply]
I can't reproduce that behavior. On this .org file with `?plain=1` <https://github.com/200ok-ch/organice/blob/master/README.org?plain=1> I can click a line number, then click the triple-dot button that appears to the left of the line number, and get a popup menu which includes the option "Reference in new issue".

In repos with Issues disabled, I still see a popup menu, but that option is gone (which makes sense). Example: https://github.com/bzg/org-mode/blob/main/README.org?plain=1

**Reply by mankoff** [reply]
You're correct. It takes a few clicks. Clicking on the line number moves that line to the top of the window (on Firefox). Then I can click the `...` to get the menu. I thought when viewing code you could access the `...` menu without having to have the text jump to the top of the screen.

**Comment by i-ky** [comment]
Duplicates #7999.
